src/buyingcatalogue/Chart.yaml

```diff
@@ -56,17 +58,20 @@ dependencies:
   version: ~0.1.0
 - name: mp
   condition: mp.enabled
-  version: ~0.1.0
+  version: ~1.25.0-alpha 
+  repository: "https://gpitfuturesdevacr.azurecr.io/helm/v1/repo"
 - name: pb
   condition: pb.enabled
-  version: ~0.1.0
+  version: ~1.25.0-alpha 
+  repository: "https://gpitfuturesdevacr.azurecr.io/helm/v1/repo"
 - name: admin
   condition: admin.enabled
-  version: ~0.1.0-alpha 
+  version: ~1.25.0-alpha 
   repository: "https://gpitfuturesdevacr.azurecr.io/helm/v1/repo"
 - name: of
   condition: of.enabled
-  version: ~0.1.0
+  version: ~1.25.0-alpha 
+  repository: "https://gpitfuturesdevacr.azurecr.io/helm/v1/repo"
```

Comment:
How about a variable for the dev ACR? Because how often is it going to change or be unique per chart?

Reply:
Good point, I'll have a look whether we can add variables / template the main Chart.yaml file